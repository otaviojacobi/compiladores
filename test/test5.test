/* Should NOT accept multiline string without line breaks*/
string a = "oie
teste";