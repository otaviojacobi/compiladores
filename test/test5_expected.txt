2 TK_PR_STRING [string]
2 TK_IDENTIFICADOR [a]
2 TK_ESPECIAL [=]
2 TOKEN_ERRO ["]
2 TK_IDENTIFICADOR [oie]
3 TK_IDENTIFICADOR [teste]
3 TOKEN_ERRO ["]
3 TK_ESPECIAL [;]